_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Exploring query enhancements
+parent: Analyzing data
+nav_order: 20
+---
+
+# Exploring query enhancements
+Introduced 2.18
+{: .label .label-purple }
+
+Starting with OpenSearch 2.17, query enhancements have been made. These enhancements are experimental and may be subject to change or instability. Enhancements include the following: 
+
+- Query languages PPL and SQL, with **Query Assist** for PPL
+- Multiline query editor for PPL and SQL and autocomplete for PPL and DQL
+- Data selector with **Data Explorer** that supports index patterns, indexes, and Amazon S3 connections and data configuration that supports selecting the appropriate query language based on the data type
```

Comment:
is it there enough to call out that index patterns aren't required for exploring your data within your indexes within OpenSearch Dashboards but still required if you want to create visualizations

Reply:
Does this communicate the appropriate message: `OpenSearch Dashboards allows for exploring your data within your indexes without using index patterns`